src/main/java/com/opencart/repository/LoginModelRepository.java

```diff
@@ -0,0 +1,18 @@
+package com.opencart.repository;
+
+import com.opencart.datamodel.LoginModel;
+
+public class LoginModelRepository {
+
+    private LoginModelRepository() {
+
+    }
+
+    public static LoginModel getLoginModel() {
+        String password = "admin";
+        String email = "test@gmail.com";
```

Comment:
You can't hardcode input data in values. Use  System.getenv() or System.getProperty() instead